### ISSUE TITLE
bodhi.get_releases() is now broken

### ISSUE THREAD

**Issue by kparal** [fedora-infra/bodhi#784]
Using `python-fedora-0.7.1-4.fc23.noarch`:

```
In [1]: import fedora.client

In [2]: bodhi =  fedora.client.BodhiClient()

In [3]: bodhi.get_releases()
---------------------------------------------------------------------------
BodhiClientException                      Traceback (most recent call last)
<ipython-input-3-edab0102d5c5> in <module>()
----> 1 bodhi.get_releases()

/usr/lib/python2.7/site-packages/fedora/client/bodhi.pyc in wrapper(*args, **kwargs)
     94         except Exception:
     95             pass
---> 96         raise BodhiClientException(problems)
     97     return wrapper
     98 

BodhiClientException: csrf_token is missing
name is missing
long_name is missing
branch is missing
id_prefix is missing
dist_tag is missing
stable_tag is missing
testing_tag is missing
candidate_tag is missing
override_tag is missing
```

This works when I downgrade to `python-fedora-0.5.5-1.fc23.noarch`.

**Comment by lmacken** [comment]
Very weird, since `curl https://bodhi.fedoraproject.org/releases/` works fine without any arguments or a CSRF token...

**Comment by ryanlerch** [comment]
fedora.client.BodhiClient() is now deprecated. however, using the replacement bodhi.client.bindings still fails, returning the following server error:

```
ServerError: ServerError(https://bodhi.fedoraproject.org/releases/, 403, Error returned from json module while processing https://bodhi.fedoraproject.org/releases/: Expecting value: line 6 column 1 (char 5) 
```

The steps to get the error are:

```
In [1]: import bodhi.client                                                                     
In [2]: client = bodhi.client.bindings.BodhiClient()
In [3]: client.get_releases()
```

**Comment by bowlofeggs** [comment]
The patch for this issue should be included in the upcoming 2.10.0 release:

https://github.com/fedora-infra/bodhi/pull/1744

**Comment by kparal** [comment]
Should it be fixed? I still see it with `bodhi-2.10.0b0`:

```
In [10]: import bodhi.client
In [11]: client = bodhi.client.bindings.BodhiClient(staging=True)
In [12]: client.get_releases()
...
ServerError: ServerError(https://bodhi.stg.fedoraproject.org/releases/, 403, Error returned from json module while processing https://bodhi.stg.fedoraproject.org/releases/: Expecting value: line 6 column 1 (char 5)
```

**Comment by bowlofeggs** [comment]
@kparal to clarify, did you mean that you tried it with the 2.10.0b0 server, but a < 2.10.0 client? The fix is in bodhi.client.bindings, so you will need a 2.10.0 client. You can get one from Copr:

http://copr.fedorainfracloud.org/coprs/bowlofeggs/bodhi-pre-release/
